translations/fr.po

```diff
@@ -39,6 +39,10 @@ msgid ""
 "Freecycle has updated its privacy policy! You can see it here: https://www."
 "freecycle.org/about/privacy \n"
 msgstr ""
+"\n"
```

Comment:
something is odd about this, i wonder if there's a setting in POedit that is creating extra linefeeds.

Reply:
In POedit, the source text is:
\n
\n
Freecycle has updated its privacy policy! You can see it here: https://www.freecycle.org/about/privacy \n

The translated text is:
\n
\n
Freecycle a mis à jour sa politique de confidentialité ! Vous pouvez le voir ici : https://www.freecycle.org/about/privacy \n

I noticed there was a carriage return at the end perhaps. Could that be it?

Reply:
There's a separate source text without the \n's and without the URL:
Freecycle has updated its privacy policy! You can see it by clicking here

The translated text is:
Freecycle a mis à jour sa Politique de Confidentialité ! Vous pouvez le voir en cliquant ici

Reply:
oh you're right, there are 2 newlines in the source. i didn't see it in the diff because github wasn't showing me those lines.  all good then, i guess!